samples/package.json

```diff
@@ -16,12 +16,13 @@
   },
   "dependencies": {
     "@google-cloud/dialogflow": "^2.0.0",
+    "@grpc/grpc-js": "^1.0.3",
```

Comment:
Why do we need that?

Reply:
Looks like I just needed to update `google-gax` so that the grpc-js dependency to ~1.0.3